front/public/locales/fr/allowances.json

```diff
@@ -11,19 +11,30 @@
   "from": "De",
   "begin_position": "Début",
   "marecoWholePath": "Marche économique (trajet complet)",
+  "sandardAllowancesWholePath": "Marges de régularité",
+  "engineeringAllowances": "Marges de construction",
   "margedTrainSchedule": "Avec marges",
   "allowanceByInterval": "Marges par intervalles",
   "ecoAllowanceByInterval": "Marches Economiques par intervalles",
   "allowanceModified": {
-    "marecoAdd": "Marche économique ajoutée",
-    "marecoDel": "Marche économique supprimée",
-    "construction": "Marche de construction modifiée"
+    "marecoAdd": "Marge ajoutée",
+    "marecoDel": "Marge supprimée",
+    "construction": "Marge modifiée"
   },
+  "specificValuesOnIntervals" : "Valeurs spécifiques sur intervales",
   "allowanceGlobalType": {
     "mareco": "Mareco",
-    "construction": "Construction"
+    "construction": "Construction",
```

Comment:
Probably not my scope to review front but i'm curious : why is there a `"construction": "Construction"` if there is a `"engineering": "Construction"` ?